popup.css

```diff
@@ -0,0 +1,88 @@
+.wrapping-container .active {
+  display: none;
+}
+
+.modal-window {
+  background: hsl(242deg 38% 27%);
+  border: 1px solid #fff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  margin: 50px 24px 0 24px;
+  display: flex;
+  position: fixed;
+  z-index: 2;
+  top: -36px;
+  overflow: hidden;
```

Comment:
- [ ] Good job implementing the popup however, the user can't scroll down in the popup, please consider fixing this issue

![Screenshot_20220512_201356](https://user-images.githubusercontent.com/25227022/168152038-4397e4a4-be42-4770-ab59-7f3b377f03de.png)

Reply:
Hello @Ismailco ,
Thank you for the review. 
I will really appreciate it if you could provide some more insight on how I can add the scroll for the modal. 
> I have tried the following:
   - Adding an "overflow-y: "visible" to the modal container class in css file
   - Adding classname.style.overflowY="auto" in the javascript file
   
But neither of the two methods are working